Dockerfile

```diff
@@ -75,36 +72,39 @@ RUN mkdir /tmp/models \
     && mv /tmp/models/v2.1_beta1/svm_models /data/ \
     && cd / && rm -r /tmp/models
 
+RUN groupadd --gid 901 acousticbrainz
+RUN useradd --create-home --shell /bin/bash --uid 901 --gid 901 acousticbrainz
+
+RUN chown acousticbrainz:acousticbrainz /code
+
 # Python dependencies
-RUN mkdir /code/docs/
-COPY docs/requirements.txt /code/docs/requirements.txt
-COPY requirements.txt /code/requirements.txt
+RUN mkdir /code/docs/ && chown acousticbrainz:acousticbrainz /code/docs/
+COPY --chown=acousticbrainz:acousticbrainz docs/requirements.txt /code/docs/requirements.txt
+COPY --chown=acousticbrainz:acousticbrainz requirements.txt /code/requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt
 
-COPY package.json /code
+COPY --chown=acousticbrainz:acousticbrainz package.json /code
+
+USER acousticbrainz
```

Comment:
Thinking out loud about this - if we don't want to create the user in the -base target, we could skip the installation of npm packages here. Instead, only install in -prod. During dev we get users to run npm install again anyway to create node_modules in their acousticbrainz-server which is bind-mounted into the container.
This allows us to keep running as root in development (and fixes the concern in the PR body).

Still a pending question about when we should chown /code (and requirements.txt) in the -prod build

Reply:
I would prefer not to add an extra step to the build instructions. In ListenBrainz, we use a different container to build npm assets automatically into the correct place, which is ideally what I'd want for AB too.